thunderbirdparser/src/org/sleuthkit/autopsy/thunderbirdparser/ThunderbirdMboxFileIngestModule.java

```diff
@@ -408,11 +443,19 @@ private BlackboardArtifact addArtifact(EmailMessage email, AbstractFile abstract
         String senderAddress;
         senderAddressList.addAll(findEmailAddresess(from));
         
-        AccountFileInstance senderAccountInstance = null;        
+        AccountFileInstance senderAccountInstance = null;
+
+        Case openCase;
+        try {
+            openCase = Case.getOpenCase();
+        } catch (NoCurrentCaseException ex) {
+            logger.log(Level.WARNING, "Exception while getting open case.", ex); //NON-NLS
+            return null; 
+        }
```

Comment:
The addArtifact method is called in a loop. The exception needs to propagate, to stop processing. If the case has been closed, there is nothing more for this module to do, and continuing to loop does no good.